mungegithub/mungers/submit-queue.go

```diff
@@ -259,9 +259,8 @@ func (sq *SubmitQueue) AddFlags(cmd *cobra.Command, config *github.Config) {
 		"kubernetes-e2e-gce-scalability",
 		"kubernetes-kubemark-5-gce",
 	}, "Comma separated list of jobs in Jenkins to use for stability testing")
-	cmd.Flags().StringSliceVar(&sq.WeakStableJobNames, "weak-stable-jobs", []string{
-		"kubernetes-kubemark-500-gce",
```

Comment:
I don't want to add the code that enables it, together with adding the test in a single PR.